### PR TITLE
Fixed typo in integration tests README.md

### PR DESCRIPTION
**Pull request by neubig**
This PR fixes a typo in the integration tests README.md.

The first draft was entirely authored and pushed to github by OpenDevin! :) For proof, see the video below:

Video:
https://drive.google.com/file/d/1qCJzKARAV-6JRd-9fRwyV5G9K7c3ytLP/view?usp=sharing

Fixes: https://github.com/OpenDevin/OpenDevin/issues/1433

### CODE REVIEW

File: tests/integration/README.md [hunk 1]
```diff
@@ -99,3 +99,4 @@ That's it, you are good to go! When you launch an integration test, mock
 responses are loaded and used to replace a real LLM, so that we get
 deterministic and consistent behavior, and most importantly, without spending real
 money.
+
```
Comment (enyst) [inline tests/integration/README.md:102]:
```suggestion
```

**Comment by enyst** [comment]
This is so fun! It will need to learn linting though, it added a line at the end of file. 😅

On second thought, maybe I shouldn't have fixed it... we could maybe explain about it in natural language?

**Review by enyst — APPROVED** [review]

File: tests/integration/README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ You can choose any model you'd like to generate the mock responses.
 You can even handcraft mock responses, especially when you would like to test the behaviour of agent for corner cases. If you use a very weak model (e.g. 8B params), chance is most agents won't be able to finish the task.
 
 ```bash
-# Remove logs iff you are okay to lose logs. This helps us locate the prompts and responses quickly, but is NOT a must.
+# Remove logs if you are okay to lose logs. This helps us locate the prompts and responses quickly, but is NOT a must.
```
Comment (li-boxuan) [inline tests/integration/README.md:83]: Interestingly, this is not a typo! 🤣
